site/plugins.json

```diff
@@ -86,7 +86,19 @@
     "name": "Visual diff (Applitools)",
     "package": "netlify-plugin-visual-diff",
     "repo": "https://github.com/applitools/netlify-plugin-visual-diff",
-    "version": "1.2.0"
+    "version": "1.5.0",
+    "compatibility": [
+      {
+        "version": "1.5.0"
+      },
+      {
+        "version": "1.4.0",
+        "nodeVersion": "<12.0.0"
+      },
+      {
+        "version": "1.2.0"
```

Comment:
It seems like this last item might not be needed, if the intent is the following: if Node <12.0.0, use `1.4.0`. Otherwise, use `1.5.0`.

Reply:
For added clarity, the items are evaluated from last to first. I.e. if a site matches a `nodeVersion` condition, it uses the related `version`. Otherwise, it tries the next version.

Reply:
thanks, i wasn't quite sure how that worked so i was over descriptive 😂  removed that version